src/components/TrackTile/hooks/useTrackers.ts

```diff
@@ -42,35 +42,71 @@ export const useTrackers = (trackersToUse?: Trackers) => {
         return;
       }
 
+      const pillarTileTrackers = installedMetrics.filter(
+        (t) => t.system === TRACKER_PILLAR_CODE_SYSTEM,
+      );
```

Comment:
Should we pass the context to this hook so we can filter by the context system and not have to have the duplicate logic?

Reply:
Sure, I'll take a look at that.

Reply:
I don't think the juice is worth the squeeze here as it results in a lot more lines of added code passing valuesContext around.